mesonbuild/modules/__init__.py

```diff
@@ -14,6 +16,36 @@ def find_program(program_name, target_name):
     _found_programs[program_name] = program
     return program
 
+
+def _get_include_args(state, include_dirs, prefix='-I'):
```

Comment:
Pass `environment` here instead of `state`. It makes it clearer what this function needs. Also, rename this to not have an `_` since that is for class-private methods, not public/shared ones.